lib/spack/spack/package.py

```diff
@@ -2319,8 +2319,13 @@ def do_activate(self, view=None, with_dependencies=True, verbose=True):
 
         extensions_layout = view.extensions_layout
 
-        extensions_layout.check_extension_conflict(
-            self.extendee_spec, self.spec)
+        try:
+            extensions_layout.check_extension_conflict(
```

Comment:
Continuation line over-indented for hanging indent (E126) for this, not sure where it needs to line up?